packages/overlay/overlay-trigger.md

```diff
@@ -38,6 +28,18 @@ When looking to leverage the `OverlayTrigger` base class as a type and/or for ex
 import { OverlayTrigger } from '@spectrum-web-components/overlay';
 ```
 
+## Description
+
+An `<overlay-trigger>` element supports the delivery of temporary overlay content based on interaction with a persistent trigger element. An element prepared to receive accessible interactions (e.g. an `<sp-button>`, or `<button>`, etc.) is addressed to `slot="trigger"`, and the content to display (either via `click` or `hover`/`focus` interactions) is addressed to `slot="click-content"` or `slot="hover-content"`, respectively. A trigger element can be linked to the delivery of content, intended for a single interaction, or both. Content addressed to `slot="hover-content"` is made available when the mouse enters or leaves the target element. Keyboard navigation will make this content available when focus enters or leaves the target element. Be thoughtful with what content you address to `slot="hover-content"`, as the content available via "hover" will be transient and non-interactive.
+
+### Placement
+
+When using the `placement` attribute of an `<overlay-trigger>` (`"top-start" | "top-end" | "bottom-start" | "bottom-end" | "right-start" | "right-end" | "left-start" | "left-end" | "none"`), you can suggest to the overlay in which direction relative to the trigger that the content should display. When there is adequate room for the content to display in the specified direction, it will do so. When adequate room is not available, the overlaid content will calculate the direction in which it has the most room to be displayed and use that direction. When setting [type](#type) to `modal`, set `placement="none"` so the underlay covers the entire viewport.
+
+### Type
```

Comment:
I’m not sure this is still current.

Reply:
Is it better now?

Reply:
Looking better!

There could be content changes in the following graph to explain https://github.com/adobe/spectrum-web-components/blob/main/packages/overlay/src/overlay-types.ts#L37, but I think that's outside of the scope of this PR.